From e371c7f140a221d4bf5d557ebabe5214ea2ff8f7 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Thu, 22 Nov 2018 14:52:06 +0800
Subject: [PATCH 5163/5242] MLK-20443 ARM64: dts: fsl-imx8mm-evk-revb: add
 touch support

commit  3b0bcb4df5d3ccf2151bac512fd070e836c2fafe from
https://source.codeaurora.org/external/imx/linux-imx.git

Add synaptics_dsx touch support for imx8mm evk revb board.

Acked-by: Andy Duan <fugang.duan@nxp.com>
Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
(cherry picked from commit 1cf1aaa3f47308f3c7eb09cd8589e3657e5d7cff)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8mm-evk-revb-rm67191.dts  |   13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-revb-rm67191.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-revb-rm67191.dts
index 986d2e2..02c2c04 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-revb-rm67191.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-revb-rm67191.dts
@@ -34,3 +34,16 @@
 		status = "okay";
 	};
 };
+
+&i2c2 {
+	synaptics_dsx_ts@20 {
+		compatible = "synaptics_dsx";
+		reg = <0x20>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&pinctrl_i2c2_synaptics_dsx_io>;
+		interrupt-parent = <&gpio1>;
+		interrupts = <9 IRQ_TYPE_LEVEL_LOW>;
+		synaptics,diagonal-rotation;
+		status = "okay";
+	};
+};
-- 
1.7.9.5

